aries_cloudagent/protocols/coordinate_mediation/v1_0/messages/inner/keylist_update_rule.py

```diff
@@ -32,7 +34,12 @@ def __init__(self, recipient_key: str, action: str, **kwargs):
 
         """
         super().__init__(**kwargs)
-        self.recipient_key = recipient_key
+        if recipient_key.startswith("did:key:"):
+            self.recipient_key = recipient_key
+        else:
+            self.recipient_key = DIDKey.from_public_key_b58(
+                recipient_key, KeyType.ED25519
+            ).did
```

Comment:
Perhaps a helper method similar to the `normalize_public_key` method would be good since we perform this normalization several times throughout the `messages.inner` package.